src/sql/workbench/contrib/notebook/browser/cellViews/textCell.css

```diff
@@ -17,6 +17,10 @@ text-cell-component .notebook-preview {
 	user-select: text;
 }
 
+.notebook-preview .rangeHighlight {
+	background-color: rgba(253, 255, 0, 0.2);
```

Comment:
Does there need to be a different highlight color for high contrast and/or dark theme? Can we check other highlight range CSS classes to follow those patterns?

Reply:
I just checked what vscode has and it does have a different color for dark theme but that's bad than what we have now. I can show you how it currently looks and we can make a decision.